docs/tooltip/tooltip.md

```diff
@@ -1,215 +0,0 @@
-
-    ![Image of elements](./assets/tooltipfit.png)
-
-    ![Image of elements](./assets/tooltipfit2.png)
-
-## API
-
-| Name | Type | Default | Required | Description |
-| -- | -- | -- | -- | -- |
-| id | string | null | no | Unique identifier |
-| className | string | null | no | Space separated list of the classes of the element |
-| style | Object | null | no | CSS styling declarations to be applied to the element |
-| children | React.ReactNode | null | no | The elements to render in the tooltip |
-| open | boolean | false | no | Whether the tooltip is shown or hidden |
-| onOpenStateChange | function | NOOP | no | Triggered when the popup is opened/closed |
-| anchor | Element | none | Yes | The element to be used as an anchor for the tooltip (will open next to it). |
-| maxHeight*** | number | 0 | no | The max height in pixels of the popup. If set to 0, the property will be ignored and receive height from its content |
-| distance*** | number | 5(px) | no | Sets the distance in pixels between the location of the tooltip and the anchor element (in the direction of the specified position property). Both negative and positive numbers accepted.* |
-| position | string literal type | "bottom" | no | Options are "bottom", "left", "right", "top" |
-| showTrigger | Array\<string> | ["mouseEnter"] | no | Shows the tooltip according to an event or events triggered on the anchor element (see [triggers](#triggers)) |
-| hideTrigger | Array\<string> | ["mouseLeave"]** | no | Hides the tooltip according to an event or events triggered on the anchor element |
-
-\* No matter the position specified by the user, a positive number will always push the tooltip away from the component, while a negative while number will push the tooltip onto the component.
-
-\** The hideTrigger will default to the opposite event/s of the triggers in the showTrigger property unless explicitly specified (focus <-> blur, mouseEnter <-> mouseLeave, click <-> click, manual <-> manual).
-
-\*** TalmiJS requirement
-
-### Triggers
-
-| Name | Event |
-| -- | -- |
-| focus | onFocus |
-| blur | onBlur |
-| mouseEnter | onMouseEnter |
-| mouseLeave | onMouseLeave |
-| click | onClick |
-| manual | When set, no other trigger applies. To be used when a trigger is either not in the list or not supported by the anchor component |
-
-## Style API
-
-It is required to supply an API which allows the user to change the arrow and the content div of the tooltip.
-
-### Internal Style
-
-You can create the arrow using several different CSS methods which produce different results. We'll start with 2 methods for styling the tooltip, the first solution will be placed in the reset style and the second in a Design System theme.
-
-The reason for creating both is to understand what we need to do in order to create a generic tooltip. In the following examples, having 4 *callout* classes (top, bottom, left, right) with their own CSS could be a viable solution.
-
-* Border (bottom) Example
-
-```html
```

Comment:
The code examples here are not stylable-specific and don't use the API that is outlined in the readme